mods/cnc/chrome/missionbrowser.yaml

```diff
@@ -24,6 +24,7 @@ Container@MISSIONBROWSER_PANEL:
 					Height: PARENT_BOTTOM - 30
 					Children:
 						ScrollItem@HEADER:
+							BaseName: scrollheader
```

Comment:
Grepping for `ScrollItem@HEADER` reveals most have a `BaseName` set, but a handful so not. Do they also need updating?

- `mods\cnc\chrome\dialogs.yaml` (2 instances)
- `mods\cnc\chrome\multiplayer-browserpanels.yaml`
- `mods\cnc\chrome\replaybrowser.yaml`

Reply:
They do not. The other mods were already  using `scrollheader` for separators, TD was an exception as the background image matched the regular scroll item artwork. But with this refactor it now needs the custom BaseName as well